### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by sdc50**
We need to create an official project email, or have some other way of reporting abuse.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at <project-email>. All
```
Comment (swainn) [inline CODE_OF_CONDUCT.md]: Use tethysplatform@gmail.com

**Comment by swainn** [comment]
@sdc50 Could we also suggest a Subject Line? Perhaps "Re: Code of Conduct" or something along those lines?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sdc50) [inline CODE_OF_CONDUCT.md]:
```suggestion
reported by contacting the project team at tethysplatform@gmail.com with subject "Re: Code of Conduct". All
```

**Comment by swainn** [comment]
@sdc50, I'm assuming the tests will pass once we merge? It's that one failing test that should be fixed in master now.

**Pull request merged**